js/checklist.js

```diff
@@ -20,7 +25,9 @@
 	 function modify_import_content_page($) {
```

Comment:
Seems like this shouldn't live in `checklist.js` but rather be a separate file, no? I guess it was like this before, though.